src/Toml/Codec/Combinator/Custom.hs

```diff
@@ -10,6 +10,7 @@ See examples below of the situations you may need the following combinators.
 
 @since 1.3.0.0
 -}
+{-# LANGUAGE LambdaCase #-}
```

Comment:
This extension is enabled in the `tomland.cabal` file by default in all modules, so no need to add it here :slightly_smiling_face: 
```suggestion
```

Reply:
Removed

Reply:
Removed